types/types.d.ts

```diff
@@ -204,6 +225,8 @@ export interface Spacetime {
   monthName(): string
   /** set the current month as a string, like 'april' */
   monthName(value: string): Spacetime
+
+  weekStart(value: number | string): Spacetime
```

Comment:
weekStart needs documentation

Reply:
Hah, serves me right for not checking my PR before opening.  Thanks, fixed.